src/functions/getIvaDefects.ts

```diff
@@ -2,11 +2,7 @@ import { Handler } from "aws-lambda";
 import { HTTPResponse } from "../models/HTTPResponse";
 import { IvaDefectsService } from "../services/ivaDefectsService";
 import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
-import {
-  EUVehicleCategory,
-  InspectionType,
-  VehicleType,
-} from "@dvsa/cvs-type-definitions/types/iva/defects/get";
+import { EUVehicleCategory } from "@dvsa/cvs-type-definitions/types/iva/defects/enums/euVehicleCategory.enum";
```

Comment:
If we are referencing this enum, should we point DefectGetIVA to this in the types package as well?